satpy/utils.py

```diff
@@ -40,6 +40,10 @@
 TRACE_LEVEL = 5
 
 
+class PerformanceWarning(Warning):
+    """Warning raised when there is a possible performance impact."""
```

Comment:
There is a similar warning class in dask, should we not use that instead?
https://docs.dask.org/en/latest/generated/dask.array.core.PerformanceWarning.html

Reply:
Pyresample also has a copy of the PerformanceWarning already in it. I'm not sure how I feel about depending on dask for a warning like this. I think in the pyresample case it was a PerformanceWarning about giving a numpy array to a dask resampler or something like that so the PerformanceWarning wasn't specifically about chunking like dask's `PerformanceWarning` says in the docstring.

In this case, it does match that description, or at least matches more than the pyresample case did.

Overall I'd prefer not to, but if you feel strongly I can do it. We may want a PerformanceWarning for other things in Satpy beyond chunking.

Reply:
Ok, just asking. We can keep this.